tests/pages/api/searchOperators.test.ts

```diff
@@ -0,0 +1,67 @@
+import { getMockRequestAndResponse } from '../../testData/mockData';
+import searchOperators, { SearchOperatorsWithErrors } from '../../../src/pages/api/searchOperators';
+import * as session from '../../../src/utils/sessions';
+import { SEARCH_OPERATOR_ATTRIBUTE } from '../../../src/constants';
+
+jest.mock('../../../src/utils/sessions.ts');
+
+describe('searchOperators', () => {
+    afterEach(() => {
+        jest.resetAllMocks();
+    });
+
+    const updateSessionAttributeSpy = jest.spyOn(session, 'updateSessionAttribute');
+
+    it('should error if the search text if entered is less than 3', () => {
+        const { req, res } = getMockRequestAndResponse({
+            body: {
+                searchText: 'abc',
```

Comment:
This test I think proves that its not working.

Reply:
changed logic so should